common/nymcoconut/src/scheme/verification.rs

```diff
@@ -291,6 +293,7 @@ mod tests {
         .unwrap();
 
         let bytes = theta.to_bytes();
+        println!("Theta (Prove credential) containing blinded message, blinded serial number, credential, zkproof (with 2 private attributes) {:?}", bytes.len());
```

Comment:
Possibly redundant print?

Reply:
Removed. Checks are already in place, so this should be OK.